test/handler/test_avra_handler.vader

```diff
@@ -0,0 +1,24 @@
+Before:
+  runtime ale_linters/avra/avra.vim
+
+After:
+  call ale#linter#Reset()
+
+Execute(The avra handler should parse errors correctly):
+  AssertEqual
+  \ [
+  \   {
+  \     'lnum': 3,
+  \     'text': "Unknown device: atmega3228p",
+  \     'type': 'E',
+  \   },
+  \   {
+  \     'lnum': 12,
+  \     'text': "Unknown directive: .EQ",
+  \     'type': 'E',
+  \   },
+  \ ],
+  \ ale_linters#avra#avra#Handle(bufnr(''), [
+  \   "main.asm(3) : Error   : Unknown device: atmega3228p"
```

Comment:
Tests are failing due to missing commas "," after each item in this list.

Reply:
I've fixed the regex. Tests should work now